src/helpers/helpers.core.js

```diff
@@ -283,22 +283,24 @@ export function _deprecated(scope, value, previous, current) {
   }
 }
 
+function indexOfDotOrLength(key, start) {
```

Comment:
The only thing I can see to make this faster would be caching on the args, but since we don't have decorators that's a pain